README.md

```diff
@@ -1,60 +1,67 @@
 <div align="center">
     <div>
         <!-- Load images from raw.githubusercontent.com to enable image rendering when viewed from https://www.npmjs.com/package/@ni/eslint-config -->
-        <img src="https://raw.githubusercontent.com/ni/javascript-styleguide/HEAD/resources/logo.svg" alt="JavaScript, TypeScript, and NI logo" width="200px">
+        <img src="https://raw.githubusercontent.com/ni/javascript-styleguide/HEAD/resources/logo.svg" alt="JavaScript, TypeScript, and NI logo" width="300" height="100">
     </div>
 </div>
 
 # NI JavaScript and TypeScript Style Guide
+
+[![NPM Version](https://img.shields.io/npm/v/@ni/eslint-config.svg)](https://www.npmjs.com/package/@ni/eslint-config)
+
 Welcome to NI's JavaScript and TypeScript linter rules for [ESLint](https://eslint.org/docs/user-guide/getting-started).
 
 ## Installation
 
 Install `@ni/eslint-config` and its peer dependencies.
 
-Use [npm view](https://docs.npmjs.com/cli/view.html) to list the correct versions of each package to install yourself.
+Use [`npm view`](https://docs.npmjs.com/cli/view.html) to list the correct versions of each package to install yourself.
 
 ```bash
 npm view @ni/eslint-config peerDependencies
 ```
 
-Alternatively, use [install-peerdeps](https://www.npmjs.com/package/install-peerdeps) as a shortcut to install the packages for you.
+Alternatively, use [`npx install-peerdeps`](https://www.npmjs.com/package/install-peerdeps) as a shortcut to install the packages for you.
 
 ```bash
 npx install-peerdeps --dev @ni/eslint-config
 ```
 
-### Angular
+### Additional Angular-specific Installation
 
-See [instructions below](#usage-angular) to use a schematic to install dependencies.
```

Comment:
I chose this anchor name, "usage-angular", specifically to avoid other Angular subsections in the future as demonstrated in another comment.

Reply:
responded in other comment